chainstate/chainstate.go

```diff
@@ -9,24 +9,44 @@ import (
 	"time"
 )
 
+type HexFormatFlag byte
+
+const (
+	RawTx HexFormatFlag = 1
+	Ef    HexFormatFlag = 2
+)
+
+func (flag HexFormatFlag) Contains(other HexFormatFlag) bool {
+	return (flag & other) == other
+}
+
 // Broadcast will attempt to broadcast a transaction using the given providers
-func (c *Client) Broadcast(ctx context.Context, id, txHex string, timeout time.Duration) (string, error) {
+func (c *Client) SupportedBroadcastFormats() HexFormatFlag {
+	switch c.ActiveProvider() {
+	case ProviderMinercraft:
+		return RawTx
+
+	case ProviderBroadcastClient:
+		return RawTx | Ef
+
+	default:
+		return RawTx
+	}
+}
+
+func (c *Client) Broadcast(ctx context.Context, id, txHex string, format HexFormatFlag, timeout time.Duration) (string, error) {
```

Comment:
`exported: exported method Client.Broadcast should have comment or be unexported (revive)go-golangci-lint`